client/src/Components/SingleGym/Modal/index.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Dialog from "@mui/material/Dialog";
+import { useFormik } from "formik";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import * as Yup from "yup";
+
+import {
+  FormControl,
+  InputLabel,
+  MenuItem,
+  OutlinedInput,
+  Select,
+} from "@mui/material";
+
+const subscriptionSchema = Yup.object().shape({
+  username: Yup.string().required("حقل اسم المستخدم مطلوب"),
+  phone: Yup.string()
```

Comment:
```suggestion
  userPhone: Yup.string()
```
as it stored in the database